ask-monty.mdx

```diff
@@ -0,0 +1,25 @@
+---
+title: "Ask Monty"
+description: "Welcome to Monterey AI, the copilot for product insights"
+---
+
+Answer any questions about your feedback like magic
+![Monty Chat](/images/chat.png)
+
+*vid or pic here*
+
+Delve deep into your feedback with "Ask Monty", an advanced GPT-4 enabled analysis tool. Pose questions regarding your feedback data, and let Monty sift through, analyze, and provide insightful answers.
+
+### **How it Works**
```

Comment:
Notice a lot of doc subtitles using both `###` as wel as `**{text}**` but, in the h3 (###) it looks like bold does not do anything additional, as the title is already bold, so can remove the extra ** in any ### line.